crates/matrix-sdk/src/notification_settings/mod.rs

```diff
@@ -282,6 +320,18 @@ impl NotificationSettings {
                         .await
                         .map_err(|_| NotificationSettingsError::UnableToUpdatePushRule)?;
                 }
+                Command::SetPushRuleActions { scope, kind, rule_id, actions } => {
+                    let request = set_pushrule_actions::v3::Request::new(
+                        scope.to_owned(),
+                        kind.to_owned(),
+                        rule_id.to_owned(),
+                        actions.to_owned(),
```

Comment:
nit: for these four, if you can pass by ownership, do it (aka remove .to_owned()), otherwise use `.clone()` to not hide the actual intent whenever possible. `to_owned()` is really for cases where you have borrowed data and want to be able to own it (from `&str` to `String` for instance, or `&UserId` to `OwnedUserId`, etc.).